app/(auth)/signup/_components/signupForm.tsx

```diff
@@ -0,0 +1,109 @@
+'use client';
+
+import { Controller, FieldValues, useForm } from 'react-hook-form';
+
+import { AxiosError } from 'axios';
+import { useRouter } from 'next/navigation';
+
+import Button from '@/components/button/button';
+import Input from '@/components/input/input';
+import { postSignUpInfo } from '@/services/api';
+
+import MemberType from './memberType';
+
+const EMAIL_REGEX = /^[A-Za-z0-9]([-_.]?[A-Za-z0-9])*@[a-z0-9]([-_.]?[A-Za-z0-9])*\.[A-Za-z]{2,3}$/;
+const PASSWORD_REGEX = /^.{8,}$/;
+
+export default function SignUpForm() {
+  const router = useRouter();
+
+  const {
+    register,
+    control,
+    handleSubmit,
+    watch,
+    formState: { errors },
+  } = useForm<FieldValues>({ mode: 'all', defaultValues: { type: 'employee' } });
```

Comment:
아마도 서비스에서 전역적으로 로그인 한 계정이 `employee`인지, `employer`인지에 따라서 달라지는 로직이 많을탠데요! 그러면 상수로 관리해볼 수 있을 것 같아요!
아래와 같은 느낌으로 표현이 될 것 같네요!
```suggestion
  } = useForm<FieldValues>({ mode: 'all', defaultValues: { type: USER_TYPE.EMPLOYEE} });
```